### PR TITLE
centroid calculated

### LINKED ISSUE #6: User's hand contours and centroid
- Draw the contours for the user's hand using the mask created based on the user's hand color.
- Also, locate the centroid of the hand.

### PR DESCRIPTION
**Pull request by Arinahalwasia**
code for calculating the centroid and changed the while loop according to that

### CODE REVIEW

File: Gesture_control.py [hunk 1]
```diff
@@ -43,6 +43,21 @@ def find_contours(thresh):
     contours,heirarchy = cv2.findContours(thresh,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE) #give list of all essential boundary points
     return contours
 
+def centroid(contours):
```
Comment (Sanchit-sk) [inline Gesture_control.py]: Pass a single contour whose centroid is to be calculated to the centroid function.
You can create another function to find the max_contour

**Review by Sanchit-sk — CHANGES_REQUESTED** [review]
Please do this minor change. Rest everything is good.

**Review by AmitSingh12345678 — CHANGES_REQUESTED** [review]
At line 73, there should be a colon after the if-condition.
At line 74, correct the indentation.
At line 72, you can rename the variables from (x,y) to (centroid_x, centroid_y).

**Review by AmitSingh12345678 — APPROVED** [review]
Seems fine to me.

**Review by PK-cod3ch3mist — APPROVED** [review]
Looks good to me

**Review by Sanchit-sk — APPROVED** [review]
Looks good to me.
Merging into Main

**Pull request merged**